starknet_py/net/account/account_client.py

```diff
@@ -179,6 +192,50 @@ async def add_transaction(
 
         return await super().add_transaction(await self._prepare_invoke_function(tx))
 
+    async def declare(
+        self,
+        compilation_source: Optional[StarknetCompilationSource] = None,
+        compiled_contract: Optional[str] = None,
+        version: int = 0,
+        search_paths: Optional[List[str]] = None,
```

Comment:
I think we should rename it to `cairo_path` so it's uniform with Compiler constructor.